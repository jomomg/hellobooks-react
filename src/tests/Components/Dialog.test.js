import React from 'react';
import DeleteDialog from '../../components/dialogs';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

describe('<DeleteDialog/>', ()=>{
    test('it renders successfully', ()=>{
        const dialog = shallow(<DeleteDialog/>);
        expect(shallowToJson(dialog)).toMatchSnapshot();
    });
});