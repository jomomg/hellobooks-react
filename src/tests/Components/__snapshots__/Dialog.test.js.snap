// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DeleteDialog/> it renders successfully 1`] = `
<div>
  <WithStyles(Dialog)
    aria-describedby="alert-dialog-description"
    aria-labelledby="alert-dialog-title"
  >
    <WithStyles(DialogTitle)
      id="alert-dialog-title"
    >
      Delete Book
    </WithStyles(DialogTitle)>
    <WithStyles(DialogContent)>
      <WithStyles(DialogContentText)
        id="alert-dialog-description"
      >
        Are You sure you want to delete this book?
      </WithStyles(DialogContentText)>
    </WithStyles(DialogContent)>
    <WithStyles(DialogActions)>
      <WithStyles(Button)
        style={
          Object {
            "backgroundColor": "green",
            "color": "white",
          }
        }
      >
        Yes
      </WithStyles(Button)>
      <WithStyles(Button)
        autoFocus={true}
        color="primary"
        style={
          Object {
            "backgroundColor": "red",
            "color": "white",
          }
        }
      >
        No
      </WithStyles(Button)>
    </WithStyles(DialogActions)>
  </WithStyles(Dialog)>
</div>
`;
